automation_testing/tests/dataview/data_view.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect } from '@playwright/test';
+import { DataviewPage } from '../../pages/DataViewPage';
+
+
+test.describe('Data View', () => {
+    let dataviewPage: DataviewPage;
+
+    test.beforeEach(async ({ page }) => {
+        dataviewPage = new DataviewPage(page);
+        await dataviewPage.visit();
+    });
+
+    test('Create new', async ({ }) => {
+        await dataviewPage.validateSnatshot(await dataviewPage.createNewBtn, 'create_new_btn');
+        await dataviewPage.createNewBtn.click();
+        await dataviewPage.validateDialogMessage('CREATE NEW');
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 8 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9498610)
```suggestion
		await dataviewPage.validateDialogMessage('CREATE NEW');
```